### DISCUSSION TITLE
GitHub Pages URL parameters and input

### DISCUSSION THREAD

**Discussion by murxg** [community/community#22002] [Other Feature Feedback, Questions, & Ideas]
<p>I have textbox input (for URLs) in <code>index.html</code> and I want to make so that you could set the value of input in URL like this: <a href="http://user.github.io/url?q=" rel="noopener nofollow ugc">user.github.io/url?q=</a> (or something like that). Is it possible?</p>

**Comment by jfullstackdev** [comment]
<p>you mean a textbox with a default text ?</p>
<p>check this out</p>
<p><a href="https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_text_defaultvalue2">https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_text_defaultvalue2</a></p>

**Comment by murxg** [comment]
<p>No, I don’t mean default value. I learned that I need <code>&lt;form&gt;</code> in my code. I did so, but it doesn’t work. I wrote <code>https://thenpcfish.github.io/index.html?ourl=https://google.com</code> and everything looked fine but in textbox (name is ourl) didn’t have <code>https://google.com</code> value.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>So you want to fill the text input element with some part of the URL? You’ll probably have to work with a bit of Javascript for that.</p>
<p>See: <a href="https://developer.mozilla.org/en-US/docs/Web/API/Document/URL" class="inline-onebox">Document.URL - Web APIs | MDN</a></p>

**Comment by murxg** [comment]
<p>Thank you! I got it working.</p>
